torchsnapshot/storage_plugin.py

```diff
@@ -60,9 +68,11 @@ def url_to_storage_plugin(url_path: str) -> StoragePlugin:
 
 
 def url_to_storage_plugin_in_event_loop(
-    url_path: str, event_loop: asyncio.AbstractEventLoop
+    url_path: str,
+    event_loop: asyncio.AbstractEventLoop,
+    storage_options: Optional[Dict[str, Any]] = None,
 ) -> StoragePlugin:
-    async def _url_to_storage_plugin(url_path: str) -> StoragePlugin:
-        return url_to_storage_plugin(url_path=url_path)
+    async def _url_to_storage_plugin() -> StoragePlugin:
+        return url_to_storage_plugin(url_path=url_path, storage_options=storage_options)
```

Comment:
A tiny fix here.

`url_path` and `storage_options` do not need to be passed to this local function since the local function can capture them too.